lms/envs/common.py

```diff
@@ -3457,6 +3465,9 @@ def _make_locale_paths(settings):
 # Maximum number of rows to fetch in XBlockUserStateClient calls. Adjust for performance
 USER_STATE_BATCH_SIZE = 5000
 
+############## Settings for Journals ######################
+JOURNALS_URL_ROOT = None
+JOURNALS_API_URL = None
```

Comment:
**Use Django App Plugin:** All of the changes in these settings files should be localized within the Journals app.

Reply:
fixed